specs/agents/log-onboarding.md

```diff
@@ -0,0 +1,71 @@
+# Log Onboarding
+
+The Agents will be a critical part of log collection onboarding for their
+application logs. This is primarily accomplished via the `log_ecs_formatting`
+configuration option, described below.
+
+## `log_ecs_formatting` configuration
+
+Configures the agent to automatically format application logs as ECS-compatible JSON
+(if possible).
+
+|----------------|---|
+| Valid options  | `on`, `off` |
```

Comment:
Didn't we land at `override` instead of `on`? I don't have strong opinions about that. Maybe @eyalkoren has?.

Reply:
You're totally right. I just got stuck on boolean again, I'll switch it.